CIFAR10/train/BAIDU-USA-GAIT_BaiduNet9_1GV100_pytorch.json

```diff
@@ -0,0 +1,27 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.1",
+	    "codeURL": "https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Training-BaiduNet9/commit/856ec31bd48cc0338ce3d31dd5299cd58b1ceaaf",
+	    "model": "BaiduNet9",
+	    "hardware": "Baidu Cloud Tesla V100*1-16GB/56 GB/12 CPU",
+	    "costPerHour": 1.0632,

```

Comment:
Can you point me to a website where I can find the price for this instance on Baidu Cloud?

Reply:
https://cloud.baidu.com/doc/GPU/Pricing.html#.E5.8C.85.E5.B9.B4.E5.8C.85.E6.9C.88.E8.AE.A1.E8.B4.B9   The price in this table is in RMB, the fifth row is the cloud instance we used in the experiments, its cost for it is 5129 RMB/month, we calculated the currency as 6.70 though recently the currency between RMB and USD changes fast to about 6.88.